### ISSUE TITLE
[Lazarus/Linux/Windows] Cannot Install Package

### ISSUE THREAD

**Issue by GregSpa** [graphics32/graphics32#113]
Hello,

I'm Delphi user since Delphi1 :)
GR32 is superb, required and very useful library. Thank you!

Installed latest GR32 on RX3 and XE6 without any problem.
Now I want to try on Linux and Lazarus. However at installing package I found some issues.
First  #20 - the changing compiler options helped.
But now: 

`GR32_Transforms.pas(1790,26) Error: identifier idents no member "Create"`

in: 1790 line (_FFocalPoint := TPointF.Create_)

```
procedure TRadialDistortionTransformation.PrepareTransform;
var
  r: TRect;
begin
  if IsRectEmpty(SrcRect) then
    raise Exception.Create(RCStrSrcRectIsEmpty);
  TransformValid := not IsRectEmpty(SrcRect);
  if Not TransformValid then
    Exit;

  // center / focal point relative to which all (un)distortions are calculated

  FFocalPoint := TPointF.Create(

    (SrcRect.Right + SrcRect.Left) / 2,
    (SrcRect.Bottom + SrcRect.Top) / 2);

  r := MakeRect(SrcRect);
  r_0 := Sqrt(2*Sqr(Min(r.Width, r.Height)))/2;

  PrepareReverseMap;
end;  
```       

What do you think? :)

**Comment by micha137** [comment]
Try FloatPoint() instead of TPointF.Create()

**Comment by andersmelander** [comment]
I have no experience with FPC but as far as I have been able to determine FPC does support record constructors. Maybe there's a conditional somewhere that prevents the TPointF constructor from being declared under FPC? (don't have time to look it up myself right now)

**Comment by GregSpa** [comment]
For the Windows 10 is the same...

**Comment by andersmelander** [comment]
Yes. Why would the OS have any influence on a compiler error?

**Comment by GregSpa** [comment]
> Try FloatPoint() instead of TPointF.Create()

It looks like FocalPoint not FloatPoint.

```
{ TPointF }
  TPointF =
{$ifndef FPC_REQUIRES_PROPER_ALIGNMENT}
  packed
{$endif FPC_REQUIRES_PROPER_ALIGNMENT}
  record
       x,y : Single;
       public
          function Add(const apt: TPoint): TPointF;
          function Add(const apt: TPointF): TPointF;
          function Distance(const apt : TPointF) : Single;
          function DotProduct(const apt : TPointF) : Single;
          function IsZero : Boolean;
          function Subtract(const apt : TPointF): TPointF;
          function Subtract(const apt : TPoint): TPointF;
          procedure SetLocation(const apt :TPointF);
          procedure SetLocation(const apt :TPoint);
          procedure SetLocation(ax,ay : Longint);
          procedure Offset(const apt :TPointF);
          procedure Offset(const apt :TPoint);
          procedure Offset(dx,dy : Longint);

          function  Scale (afactor:Single)  : TPointF;
          function  Ceiling : TPoint;
          function  Truncate: TPoint;
          function  Floor   : TPoint;
          function  Round   : TPoint;
          function  Length  : Single;
          class operator = (const apt1, apt2 : TPointF) : Boolean;
          class operator <> (const apt1, apt2 : TPointF): Boolean;
          class operator + (const apt1, apt2 : TPointF): TPointF;
          class operator - (const apt1, apt2 : TPointF): TPointF;
          class operator - (const apt1 : TPointF): TPointF;
          class operator * (const apt1, apt2: TPointF): Single; // scalar product
          class operator * (const apt1: TPointF; afactor: single): TPointF;
          class operator * (afactor: single; const apt1: TPointF): TPointF;
       end;
  { TRectF }   
```

**Comment by GregSpa** [comment]
#114 - OK. Fixed and installed in Lazarus successfully

**Comment by GregSpa** [comment]
> Yes. Why would the OS have any influence on a compiler error?

Maybe for compiler not (if is well programmed). But in practice there is plenty dependencies may fail.
I work on Windows/MacOS/Linux daily. Very rarely all the same things work for each of them from the first kick.

**Comment by micha137** [comment]
I would just merge #114...

**Comment by micha137** [comment]
Anything else that stops FPC?

**Comment by micha137** [comment]
Should be resolved together with #121 now.
